packages/cli/src/LoadNodesAndCredentials.ts

```diff
@@ -110,31 +115,23 @@ export class LoadNodesAndCredentials implements INodesAndCredentials {
 	}
 
 	private async loadNodesFromBasePackages() {
-		const nodeModulesPath = await this.getNodeModulesPath();
-		const nodePackagePaths = await this.getN8nNodePackages(nodeModulesPath);
-
-		for (const packagePath of nodePackagePaths) {
-			await this.runDirectoryLoader(LazyPackageDirectoryLoader, packagePath);
-		}
+		await this.runDirectoryLoader(LazyPackageDirectoryLoader, NODES_BASE_DIR);
```

Comment:
This change will break custom nodes installed in `node_modules`. But that's something I'd like us to do anyways, as letting users install their own packages in our `node_modules` can mess with n8n dependencies. Maybe this should wait until 1.0 release.